contracts/apps/disputable/examples/RegistryApp.sol

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
Perhaps let's move this to a top level `examples/` folder

Reply:
I like the idea! Addressed now, let me know your thoughts :)

Reply:
Ah, I was actually meaning even outside of the `contracts/` folder! Do you think it would still be worth leaving it in the `contracts/examples` folder?

Reply:
Sounds better!